src/javascript/_common/base/client_base.js

```diff
@@ -386,8 +386,8 @@ const ClientBase = (() => {
 
     const hasSvgAccount = () => !!(getAllLoginids().find(loginid => /^CR/.test(loginid)));
 
-    const canChangeCurrency = (statement, mt5_login_list, is_current = true) => {
-        const currency             = get('currency');
+    const canChangeCurrency = (statement, mt5_login_list, loginid, is_current = true) => {
```

Comment:
Please also update all the users of this function, I've seen this is being used in `metatrader.js` file

Reply:
I updated function by adding `loginid`  as possible argument. If there is no `loginid` its not ruined the logic of using this function.